lib/swaps/SwapClientManager.ts

```diff
@@ -132,23 +133,36 @@ class SwapClientManager extends EventEmitter {
 
     // loop through swap clients to find locked lnd clients
     const initWalletPromises: Promise<any>[] = [];
-    const createdLndWallets: string[] = [];
+    const initializedLndWallets: string[] = [];
+    let initializedRaiden = false;
     for (const swapClient of this.swapClients.values()) {
       if (isLndClient(swapClient) && swapClient.isWaitingUnlock()) {
         const initWalletPromise = swapClient.initWallet(walletPassword, seedMnemonic).then(() => {
-          createdLndWallets.push(swapClient.currency);
+          initializedLndWallets.push(swapClient.currency);
         }).catch((err) => {
           swapClient.logger.debug(`could not initialize wallet: ${err.message}`);
         });
         initWalletPromises.push(initWalletPromise);
       }
     }
 
-    await Promise.all(initWalletPromises);
+    if (!this.raidenClient.isDisabled()) {
+      const { keystorepath } = this.config.raiden;
+      const keystorePromise = seedutil(seedMnemonic, walletPassword, keystorepath).then(() => {
```

Comment:
In order to programmatically unlock raiden we'll need to store the password of the keystore file in plain text. If I'm reading the code correctly it's using the xud master password to protect the keystore. Can we temporarily set it to empty string until we have a way of unlocking raiden via the REST API?

Reply:
Good thinking, I will do this on a temporary basis until we get a much needed call to unlock the keystore.

Reply:
Done, I had to make a few changes to the seedutil tool to make the encryption password optional.